src/System.Xml.ReaderWriter/src/System/Xml/AsyncHelper.cs

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace System.Xml
+{
+    internal static class AsyncHelper
+    {
+        public static readonly Task DoneTask = Task.FromResult(true);
```

Comment:
In general I assume that when a PR is put up, the intent is for all files to be reviewed.  Moving forward, I recommend that if the intent is only for certain portions to be reviewed that such information be included in the PR description.  Thanks.